src/components/FormikAutocomplete.jsx

```diff
@@ -1,20 +1,47 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { getIn } from 'formik';
 import { Autocomplete, InputLabel, TextField } from '@mui/material';
 import PropTypes from 'prop-types';
 
 /* eslint-disable react/jsx-props-no-spreading */
 function FormikAutocomplete(props) {
-  const { label, id, labelStyle, formik, placeholder, ...otherProps } = props;
+  const {
+    label,
+    id,
+    labelStyle,
+    formik,
+    placeholder,
+    onChange,
+    ...otherProps
+  } = props;
   const values = getIn(formik.values, id, []);
+
+  const [inputValue, setInputValue] = useState('');
+
   return (
     <>
       <InputLabel sx={labelStyle}>{label}</InputLabel>
       <Autocomplete
         id={id}
         multiple
         value={values}
-        onChange={(e, val) => formik.setFieldValue(id, val)}
+        inputValue={inputValue}
```

Comment:
`inputValue` is the current text that the user is typing. It does not include the `values` already chosen.